Gemfile

```diff
@@ -3,11 +3,19 @@ source 'https://rubygems.org'
 group :test do
   gem 'rake'
 
-  puppetversion = ENV.key?('PUPPET_VERSION') ? "#{ENV['PUPPET_VERSION']}" : ['>= 3.0.0','< 4.0']
+  puppetversion = if ENV['PUPPET_VERSION']
+                    ENV['PUPPET_VERSION'].to_s
```

Comment:
Is it possible for this to not be a string?

Also, the check should probably be `if ENV['PUPPET_VERSION'].nil?` (and the cases swapped around accordingly) to be a little more explicit.